src/components/Inputs/TextMaskedInput/TextMaskedInput.js

```diff
@@ -84,7 +97,7 @@ export const TextMaskedInput = (props) => {
         keepCharPositions={restProps.keepCharPositions}
         pipe={restProps.pipe}
       />
-      {getError()}
+      {!doValidation && getError(currentError, formTouched)} 
```

Comment:
`doValidation` indicates the `TextMaskedInput` consumer is doing their own validation which is how birthdate and zipcode use it. The only really want the mask rendering but not its validation (they do their own)